core/src/main/java/com/linecorp/armeria/common/AggregatedHttpRequest.java

```diff
@@ -158,18 +158,18 @@ static AggregatedHttpRequest of(HttpMethod method, String path, MediaType mediaT
      * @param path the path of the request
      * @param mediaType the {@link MediaType} of the request content
      * @param content the content of the request
-     * @param trailingHeaders the trailing HTTP headers
+     * @param trailers the trailing HTTP headers
      */
     static AggregatedHttpRequest of(HttpMethod method, String path, MediaType mediaType,
-                                    HttpData content, HttpHeaders trailingHeaders) {
+                                    HttpData content, HttpHeaders trailers) {
```

Comment:
Didn't change all the parameters and variables names to `trailers`, but just a few classes related `HttpRequest` and `HttpResponse`.

Reply:
Changed my mind. Rename all parameters and variables. :-)